umbraco/Umbraco-CMS
# AB#9725 - Removed IUmbracoApplicationLifetimeManager

Fixes [AB#9725](https://dev.azure.com/umbraco/243e7927-03b2-44e2-908f-d4ac7ea5daaa/_workitems/edit/9725)

### Details
- Removed `IUmbracoApplicationLifetimeManager`
- Removed `IUmbracoApplicationLifetime.ApplicationInit`
- Removed `AspNetCoreComposer` + `AspNetCoreComponent`
- Migrated `WebProfilerComposer` + `WebProfilerComponent`
  - Added to UmbracoBuilder + Created new `InitializeWebProfiling` notification hander executed on `UmbracoApplicationStarting`
- Fixed issues with web profiler rendering on the frontend (This can't be handled in the same way anymore)

### Test
- Run in debug mode
- Visit https://localhost:44331/?umbdebug=true
  - First request should include the boot profiling
  - Subsequent request should only include the request profiling
